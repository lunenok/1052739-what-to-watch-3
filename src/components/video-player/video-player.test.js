import React from "react";
import renderer from "react-test-renderer";
import VideoPlayer from "./video-player.jsx";

const testMovie = {
  id: 1,
  title: `Bohemian Rhapsody`,
  genre: `Drama`,
  year: 2019,
  imgSrc: `war-of-the-worlds.jpg`,
  videoSrc: `https://upload.wikimedia.org/wikipedia/commons/transcoded/b/b3/Big_Buck_Bunny_Trailer_400p.ogv/Big_Buck_Bunny_Trailer_400p.ogv.360p.webm`
};

it(`Should video player render correctrly`, () => {
  const {videoSrc, imgSrc} = testMovie;
  const tree = renderer
    .create(
        <VideoPlayer
          videoSrc={videoSrc}
          imgSrc={imgSrc}
          isPlaying={true}
        />, {
          createNodeMock: () => {
            return {};
          }
        }).toJSON();

  expect(tree).toMatchSnapshot();
});
