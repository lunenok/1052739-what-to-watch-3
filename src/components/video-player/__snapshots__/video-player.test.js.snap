// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Should video player render correctrly 1`] = `
<video
  height={175}
  muted={true}
  width={280}
/>
`;
